johnnycanencrypt/__init__.py

```diff
@@ -341,6 +341,63 @@ def __contains__(self, other: Union[str, Key]) -> bool:
             return False
         return False
 
+    def update_expiry_in_subkeys(
+        self, key: Key, subkeys: List[str], expiration: datetime, password: str
+    ) -> Key:
+        """Adds a new user id to the given key, saves on the database. Then returns the modified key object
```

Comment:
@kushaldas shouldn't this be "Updates the expiration to the given key, saves on the database"?